### PR TITLE
PRTL-2659 - descriptive button names (a11y)

### PR DESCRIPTION
**Pull request by incakoala**
# Jira: 
[PRTL-2658 ](https://clever.atlassian.net/browse/PRTL-2658)
[PRTL-2659 ](https://clever.atlassian.net/browse/PRTL-2659)

# Overview:
Added ariaLabel to the Reply and Show more/less button components that are used in the announcements in messaging

# Screenshots/GIFs:
Tested with screen reader and made sure it picked up the descriptions

Screen Shot 2022-07-20 at 3 36 39 PM

Screen Shot 2022-07-20 at 6 10 19 PM

Screen Shot 2022-07-20 at 6 10 06 PM

# Testing:

- [x] Unit tests (make unit-test)
- [x] make format-all
- [x] make lint
- Manual tests:
  - [x] Chrome
  - [x] Safari
  - [ ] IE11

# Roll Out:

- Before merging:
  - [ ] [ not needed ] Updated docs
  - [ ] [ not needed ] Bumped version in `package.json`
    - Breaking change?
      - If it is a beta component run `npm version minor`
      - If the component is not in beta run `npm version major`
    - New component or backward-compatible component feature change? Run `npm version minor`
    - Only changing documentation? All good. Skip this step.
  - [ not needed ] After creating a new component, make sure to add it to the Components List in `ComponentsView.jsx`. To do so:
    - [ ] Add a screenshot of the component in `docs/assets/img` with the format `<COMPONENT URL LINK>.png`
- After merging:
  - [ ] [ not needed ] Deployed updated docs (`make deploy-docs`)
  - [ ] [ not needed ] Posted in #eng if I made a breaking change to a beta component

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.185.0",
+  "version": "2.186.0",
```
Comment (Maddiebusta) [inline package.json:3]: i think this should stay at 2.185. in general you dont need to add package.json

**Review by Maddiebusta — CHANGES_REQUESTED** [review]
remove package.json and you should be good to go!

**Review by Maddiebusta — APPROVED** [review]
good to go!

**Comment by zohnnyclever** [comment]
@incakoala one problem with this is that you need to update the version in `package.json` by running `npm version minor`. Otherwise, other code that uses this repo won't be able to specify the correct version that has this change.

**Pull request merged**